docs/guides/getting-started/getting-started-scaling-and-tuning.adoc

```diff
@@ -0,0 +1,62 @@
+title="Scaling"
+summary="Get started with {project_name} scaling and tuning">
+
+Once you have {project_name} started you can next consider adapting your instance to whatever load may be needed. Scaling and tuning can help you address:
+- minimizing resource utilization
+- excessive response times
+- database pool contention
+- out of memory errors, or excessive garbage collection overhead
+- providing higher availability via horizontal scaling
+
+== Vertical Scaling
+
+As you monitor your {project_name} workload if you see that CPU or memory is either under or over utilized, please consult <@links.ha id="concepts-memory-and-cpu-sizing" /> to better tune the resources available to the VM.
+
+Before increasing the amount of memory available to the VM, in particular when experiencing an out of memory error, it's best to determine what is contributing to the increased footprint via a heap dump. Excessive response times may also indicate the HTTP work queue is too large and tuning for load shedding would be better than simply providing more memory - see below.
+
+=== VM Tuning
+
+JVM Tuning is highly related to the resources available. For example the default garbage collection settings may work well for nominal heap sizes, but may be not ideal for much smaller or larger VMs. If you are experiencing excessive GC overhead in addition to utilizing the JVM GC tuning options, please open an issue or discussion to raise awareness of your situation.
+
+While there is some automatic adjustment based upon how many cores you make available to your {project_name} server, fine-tuning thread counts can improve overall throughput see <@links.ha id="concepts-threads" /> - but this must be done in conjunction with other VM resources, such as database connections - see <@links.ha id="concepts-database-connections" /> otherwise you may be moving a bottleneck somewhere else.
+
+To limit memory utilization of queued work and to provide for load shedding see <@links.ha id="concepts-threads#load-shedding" /> 
+
+If you are experiencing timeouts in obtaining database connections, you should consider increasing the number of connections available - see <@links.ha id="concepts-database-connections" />.
+
+=== Vertical Autoscaling
+
+Some platforms, such as Kubernetes, provide mechanisms to vertically autoscale. That is not recommended for {project_name} if it requires restarting the server instance - which is currently the case for Java on Kubernetes. You can consider instead providing higher CPU and/or memory limits to your container to allow your JVM to adapt within those limits as needed.
+
+== Horizontal Scaling
+
+A single {project_name} instance is susceptible to availability issues. If the instance goes down you will be faced with a full outage, and until the persistent sessions feature is available all of your on-going sessions will be lost. By running at least two cluster members on different machines you will greatly increase the availability of {project_name}. 
```

Comment:
As the persistent sessions feature is available as a preview feature, should we say something like "and unless the `persistent-user-sessions` feature is enabled, all sessions will be lost."

Reply:
Ok, updated the text.